profiling/src/lib.rs

```diff
@@ -140,8 +124,16 @@ static mut PREV_EXECUTE_INTERNAL: MaybeUninit<
  * Be careful out there!
  */
 extern "C" fn minit(r#type: c_int, module_number: c_int) -> ZendResult {
+    /* When developing the extension, it's useful to see log messages that
+     * occur before the user can configure the log level. However, if we
+     * initialized the logger here unconditionally then they'd have no way to
+     * hide these messages. That's why it's done only for debug builds.
+     */
     #[cfg(debug_assertions)]
-    trace!("MINIT({}, {})", r#type, module_number);
+    {
+        logging::log_init(LevelFilter::Trace);
```

Comment:
The alternative is doing an early direct call to `libc::getenv()` for logging? And later on the proper sapi_getenv+libc::getenv() fallback is used to cover other cases too?